romancal/pipeline/mosaic_pipeline.py

```diff
@@ -86,19 +87,24 @@ def process(self, input):
             product_name = input.asn["products"][0]["name"]
             try:
                 skycell_name = input.asn["target"]
-            except IndexError:
+            except KeyError:
                 skycell_name = ""
             skycell_record = []
 
-            # if this is a valid skycell name load the database and get the skycell record
+            # if this is a valid skycell name get the skycell record
             if re.match(r"r\d{3}\w{2}\d{2}x\d{2}y\d{2}", skycell_name):
-                if patch_match.PATCH_TABLE is None:
-                    patch_match.load_patch_table()
-                if patch_match.PATCH_TABLE is None:
-                    raise RuntimeError("No patch table has been loaded")
-                skycell_record = patch_match.PATCH_TABLE[
-                    np.where(patch_match.PATCH_TABLE["name"][:] == skycell_name)[0][0]
-                ]
+                # check to see if the skycell coords are in the asn header if
+                # so read the string and convert to a dictionary to match the patch table
+                try:
+                    skycell_record = json.loads(input.asn["skycell_wcs_info"])
```

Comment:
Is there an example association file with this input that you would share? Looking at this line of code makes me think this is expecting a json encoded object stored as a string inside the asn. Is that correct? If so, what's the reason for json encoding the `skycell_wcs_info` (instead of just allowing it to be encoded along with the other structured content of the asn)?

Reply:
You can create an association using skycell_asn and any level 2 file.

Reply:
I'm assuming that means there is no example association. If I generate one with `skycell_asn` it's garbled which confirms the double json encoding I asked about above. Since this PR is merged I opened a ticket for the issue: https://jira.stsci.edu/browse/RCAL-992